SimPEG/VRM/RxVRM.py

```diff
@@ -5,12 +5,15 @@
 # BASE RECEIVER CLASS FOR VRM
 #########################################
 
+
 class BaseRxVRM(Survey.BaseRx, properties.HasProperties):
     """Base VRM receiver class"""
 
     def __init__(self, locs, **kwargs):
 
-        super(BaseRxVRM, self).__init__(locs, 'None', storeProjections=False, **kwargs)
+        super(BaseRxVRM, self).__init__(
+            locs, 'None', storeProjections=False, **kwargs
+            )
```

Comment:
This is nit-picky, but would you mind closing the bracket at the same level as the `super` (eg back one indentation)